code/game/objects/items/flamethrower.dm

```diff
@@ -239,6 +239,7 @@
 	if(damage && attack_type == PROJECTILE_ATTACK && P.damage_type != STAMINA && prob(15))
 		owner.visible_message("<span class='danger'>\The [attack_text] hits the fueltank on [owner]'s [name], rupturing it! What a shot!</span>")
 		var/target_turf = get_turf(owner)
+		log_game("A fueltank held by [owner] was detonated by a proectile hit")
```

Comment:
proectile

Reply:
proyectile

Reply:
The logging also seems a bit odd. Is there really anything wrong with being more specific here?
"The fuel tank of [name] detonated from getting hit by [hitby]. Holder: [owner] ([owner.real_name])"

Reply:
im lazy